parachain/pallets/ethereum-beacon-client/src/impls.rs

```diff
@@ -71,7 +73,7 @@ impl<T: Config> Verifier for Pallet<T> {
 
 impl<T: Config> Pallet<T> {
 	/// Verifies that the receipt encoded in `proof.data` is included in the block given by
-	/// `proof.block_hash`. Inclusion is only recognized if the block has been finalized.
```

Comment:
I think this isn't true on the `verify_receipt_inclusion` method anymore. Finalization checks are handled by the beacon client.